scss/_patterns_footer.scss

```diff
@@ -3,13 +3,15 @@
 // Footer pattern styling
 @mixin vf-p-footer {
   .p-footer {
-    border-top: 1px dotted $color-x-dark;
-    font-size: $sp-small;
+    border-top: 1px solid $color-mid-light;
+    color: $color-dark;
```

Comment:
We need to be careful when setting this color as we are not setting the background color. As this is currently the body color anyway I would suggest removing it.